extensions/amp-bind/amp-bind.md

```diff
@@ -76,48 +141,76 @@ The state can be initialized with the `amp-state` component:
+A **binding** is a special attribute of the form `[property]` that links an element's property to an [expression](#expressions).
 
 When the **state** changes, expressions are re-evaluated and the bound elements' properties are updated with the new expression results.
 
 `amp-bind` supports data bindings on three types of element state:
 
-| Type | Syntax | Details |
+| Type | Attribute | Details |
 | --- | --- | --- |
 | [Node.textContent](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent) | `[text]` | Supported on most text elements.
-| [CSS Classes](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/class) | `[class]` | Expression result must be a space-delimited string.
-| Element-specific attribute | `[<attr>]` | Only whitelisted attributes are supported.<br>Boolean expression results toggle boolean attributes.
+| [CSS classes](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/class) | `[class]` | Expression result must be a space-delimited string.
+| Size of [AMP elements](https://www.ampproject.org/docs/reference/components) | `[width]`<br>`[height]` | Changes the width and/or height of the AMP element. |
+| Element-specific attribute | [Various](#element-specific-attributes). | Boolean expression results toggle boolean attributes.
 
-- For security reasons, binding to `innerHTML` is disallowed
-- All attribute bindings are sanitized for unsafe URL protocols (e.g. `javascript:`)
+- For security reasons, binding to `innerHTML` is disallowed.
+- All attribute bindings are sanitized for unsafe values, e.g. URL protocols (e.g. `javascript:`).
 
 #### Element-specific attributes
 
-Most attributes accepted by the [AMP Validator](https://validator.ampproject.org/) for AMP and non-AMP elements, are bindable.
+Only binding to the following components and attributes are allowed. Most bindable attributes correspond to a non-bindable counterpart, e.g. for `<amp-img>`, `[src]` changes the value of `src`.
 
-There are also special bindable attributes without non-bindable counterparts:
+(*) Denotes bindable attributes that don't have a non-bindable counterpart.
 
-| Component | Attribute | Details | Example |
-| --- | --- | --- | --- |
-| amp-carousel[type=slides] | `[slide]` | The currently displayed slide index. | [Linked carousels](https://ampbyexample.com/advanced/image_galleries_with_amp-carousel/#linking-carousels-with-amp-bind)
-| amp-selector | `[selected]` | The `option` attribute values of the currently selected children elements. | [Linked carousels](https://ampbyexample.com/advanced/image_galleries_with_amp-carousel/#linking-carousels-with-amp-bind)
+| Component | Attribute(s) | Behavior |
+| --- | --- | --- |
+| `<amp-brightcove>` | `[data-account]`<br>`[data-embed]`<br>`[data-player]`<br>`[data-player-id]`<br>`[data-playlist-id]`<br>`[data-video-id]` | Changes the displayed Brightcove video. |
```

Comment:
Is this behavior accurate to changing any of these attributes? Maybe it's worth just linking to the amp-brightcove page on ampproject like you do elsewhere?